src/main/java/dev/thedocruby/resounding/mixin/SoundSystemMixin.java

```diff
@@ -33,20 +33,20 @@ public class SoundSystemMixin {
 
     @Inject(method = "play(Lnet/minecraft/client/sound/SoundInstance;)V", at = @At(value = "FIELD", target = "net/minecraft/client/sound/SoundSystem.sounds : Lcom/google/common/collect/Multimap;"), locals = LocalCapture.CAPTURE_FAILHARD)
     private void soundInfoYeeter(SoundInstance sound, CallbackInfo ci, WeightedSoundSet weightedSoundSet, Identifier identifier, Sound sound2, float f, float g, SoundCategory soundCategory){
-        if (ResoundingEngine.isOff) return;
-        ResoundingEngine.updateYeetedSoundInfo(sound, this.listener); // TODO: do this better maybe
+        if (Engine.isOff) return;
+        Engine.updateYeetedSoundInfo(sound, this.listener); // TODO: do this better maybe
     }
 
     @Inject(method = "tick()V", at = @At(value = "HEAD"))
-    private void ticker(CallbackInfo ci){ AirEffects.updateSmoothedRain(); }
+    private void ticker(CallbackInfo ci){ Air.updateSmoothedRain(); }
```

Comment:
this entire mixin shopuld be commented out with the `Air` class, as should the import above.

Reply:
Done.